site/page.js

```diff
@@ -29,7 +29,7 @@ class Page extends React.Component {
         <div className="overflow-auto h-viewport-full-mm w180-mm fixed-mm top left flex-parent-mm flex-parent--stretch-cross-mm">
           <Navigation />
         </div>
-        <div className="ml180-mm w-full px24 w-11/12-mm wmax1200-mm px60-mm mb60 mt24">
+        <div className="ml180-mm mx-auto px24 w-full w-auto-mm w-5/6-ml w-11/12-mxl wmax1200-mm px60-mm mb60 mt24">
```

Comment:
This change still results in horizontal scroll on most screen sizes. I am pretty sure you either need to more heavily restructure things, or you need to add -wmax percentage classes.

Reply:
Hm I was just looking at the implementation before this PR, and it looks like we don't use a proportionally scaled limiter there either: we force `width: auto` with `.w-auto` - otherwise main branch also scrolls!

Reply:
oh! it looks like the previous markup was using `w-auto` which was overriding the width rules in `limiter`. Dropping those width rules and favoring `w-auto` should get us there no?